### PR TITLE
Update the marketplace readme to match product features

### PR DESCRIPTION
**Pull request by taylorsperry**
Updates the readme with the latest product features. This is intended to be included in the 1.0.0 release.

- [ ] Confirm when to merge this due to mentions of Cody Pro.

## Test plan

- Previewed the readme

### CODE REVIEW

File: vscode/README.md [hunk 1]
```diff
@@ -23,29 +23,36 @@ For example, you can ask Cody:
...
 
 <!-- prettier-ignore: Uses <img> so we can fix width to 480px so all images are consistent width and look sharp @2x -->
-<img src="https://storage.googleapis.com/sourcegraph-assets/website/Product%20Animations/GIFS/Explain_Code-Sept23-Sm.gif" width="480" alt="Explain Code command">
+<img src="https://storage.googleapis.com/sourcegraph-assets/blog/vs-code-onboarding-walkthrough-dec-2023-explain.gif" width="480" alt="Explain Code command">
 
-## Custom Commands
+## Custom Commands (Beta)
 
-There's also experimental support for adding your own custom commands, defined as JSON within your repository:
+You can also build your own [Custom Commands (Beta)](https://docs.sourcegraph.com/cody/capabilities/commands#custom-commands) to tailor Cody to your workflow. Custom Commands are defined as JSON within your repository and can be saved to your workspace for your teammates to reuse.
```
Comment (taylorsperry) [inline vscode/README.md:37]: @kalanchan will this link to the docs still work with the docs update (https://docs.sourcegraph.com/cody/capabilities/commands#custom-commands)?

Comment (kalanchan) [reply vscode/README.md:37]:
```suggestion
You can also build your own [Custom Commands (Beta)](https://sourcegraph.com/docs/cody/capabilities/commands#custom-commands) to tailor Cody to your workflow. Custom Commands are defined as JSON within your repository and can be saved to your workspace for your teammates to reuse.   
```

nope, good catch! we'll have to change all docs to the new URL

File: vscode/README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 # AI that knows your entire codebase
 
-[Cody](https://about.sourcegraph.com/cody?utm_source=marketplace.visualstudio.com&utm_medium=referral) is a free and [open-source](https://github.com/sourcegraph/cody) AI coding assistant that can write, understand and fix your code. Cody is powered by Sourcegraph’s code graph, and has knowledge of your entire codebase. Install Cody to get started with free AI-powered autocomplete, chat, commands, and more.
+[Cody](https://about.sourcegraph.com/cody?utm_source=marketplace.visualstudio.com&utm_medium=referral) is a free and open core AI coding assistant that can write, understand, fix, and find your code. Cody is powered by Sourcegraph’s code graph, and has knowledge of your entire codebase. Install Cody to get started with free AI-powered autocomplete, chat, commands, and more.
```
Comment (kalanchan) [inline vscode/README.md:3]:
```suggestion
[Cody](https://about.sourcegraph.com/cody?utm_source=marketplace.visualstudio.com&utm_medium=referral) is a free AI coding assistant that can write, understand, fix, and find your code. Cody is powered by Sourcegraph’s code graph, and has knowledge of your entire codebase. Install Cody to get started with free AI-powered autocomplete, chat, commands, and more.
```

Maedah doesn't reference open core anymore, we should check with marketing on whether or not we want to call it open core and align on all docs/messaging

https://github.com/sourcegraph/sourcegraph/pull/58903/files

File: vscode/README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
 ## Autocomplete
 
-Cody autocompletes single lines, or whole functions, in any programming language, configuration file, or documentation. It’s powered by latest instant LLM models, for accuracy and performance.
+Cody autocompletes single lines, or whole functions, in any programming language, configuration file, or documentation. It’s powered by the latest instant LLM models, for accuracy and performance.
 
 <!-- prettier-ignore: Uses <img> so we can fix width to 480px so all images are consistent width and look sharp @2x -->
-<img src="https://storage.googleapis.com/sourcegraph-assets/website/Product%20Animations/GIFS/cody-completions-may2023-optim.gif" width="480" alt="Cody autocomplete">
+<img src="https://storage.googleapis.com/sourcegraph-assets/blog/vs-code-onboarding-walkthrough-dec-2023-cody-autocomplete-tsx.gif" width="480" alt="Cody autocomplete">
```
Comment (kalanchan) [inline vscode/README.md:10]: nice gif!

File: vscode/README.md [hunk 4]
```diff
@@ -61,27 +68,16 @@ For example:
...
-Embeddings for free Cody users are generated via the [Cody desktop app](https://docs.sourcegraph.com/app?utm_source=marketplace.visualstudio.com&utm_medium=referral). For Cody Enterprise customers the embeddings are generated by your Sourcegraph Enterprise instance.
-
 ## Cody Enterprise
 
 Cody Enterprise requires the use of a Sourcegraph Enterprise instance, and gives you access to AI coding tools across your entire organization. [Contact us](https://about.sourcegraph.com/contact/request-info?utm_source=marketplace.visualstudio.com&utm_medium=referral) to set up a trial of Cody Enterprise. If you’re an existing Sourcegraph Enterprise customer, contact your technical advisor.
 
 ## Feedback
 
-- [Issue tracker](https://github.com/sourcegraph/cody/issues)
-- [Discussions](https://github.com/sourcegraph/cody/discussions)
```
Comment (kalanchan) [inline vscode/README.md:77]: might want to add this back incase users have PLG related troubles

**Review by kalanchan — APPROVED** [review]
made some comments but otherwise looks good!

File: vscode/README.md (hunk 4 above)
Comment (toolmantim) [reply vscode/README.md:77]: I don't think that's clear to the user where they should go then. We also have Discord. Maybe let's keep that out.

**Review by toolmantim — APPROVED** [review]

**Pull request merged**